bower.json

```
@@ -18,19 +18,16 @@
 		"tests"
 	],
 	"dependencies": {
-		"alphaville-ui": "Financial-Times/alphaville-ui#^2.6.0",
-		"dom-delegate": "ftdomdelegate#^2.0.0",
+		"alphaville-ui": "Financial-Times/alphaville-ui#fb1f866",
```

For when [this is released](https://github.com/Financial-Times/alphaville-ui/pull/18):
```suggestion
		"alphaville-ui": "^4.0.0",
```